user.sql

```diff
@@ -0,0 +1,8 @@
+//création de la base de données User
+create DATABASE Trott;
```

Comment:
CREATE DATABASE
create database

pick one ;)

don't use uppercase letters for database, table or column names